mods/d2k/mod.yaml

```diff
@@ -215,3 +199,1986 @@ GameSpeeds:
+						Offset: 19894729
+						Length: 13368
+					^Content/d2k/GAMESFX/H_ICONF3.AUD:
+						Offset: 19908097
+						Length: 15612
+					^Content/d2k/GAMESFX/H_ISEL1.AUD:
+						Offset: 19923709
+						Length: 8279
+					^Content/d2k/GAMESFX/A_ISEL2.AUD:
+						Offset: 19931988
+						Length: 11383
+					^Content/d2k/GAMESFX/AI_RUN.AUD:
+						Offset: 19943371
+						Length: 18511
+					^Content/d2k/GAMESFX/HI_SILOS.AUD:
+						Offset: 19961882
+						Length: 14244
+					^Content/d2k/GAMESFX/HI_MAP1A.AUD:
+						Offset: 19976126
+						Length: 24075
+					^Content/d2k/GAMESFX/HI_MAP1B.AUD:
+						Offset: 20000201
+						Length: 26716
+					^Content/d2k/GAMESFX/HI_MAP1C.AUD:
+						Offset: 20026917
+						Length: 21716
+					^Content/d2k/GAMESFX/HI_MAP2A.AUD:
+						Offset: 20048633
+						Length: 32417
+					^Content/d2k/GAMESFX/HI_MAP2B.AUD:
+						Offset: 20081050
+						Length: 33724
+					^Content/d2k/GAMESFX/HI_MAP2C.AUD:
+						Offset: 20114774
+						Length: 21828
+					^Content/d2k/GAMESFX/HI_MAP3A.AUD:
+						Offset: 20136602
+						Length: 30404
+					^Content/d2k/GAMESFX/O_ISEL3.AUD:
+						Offset: 20167006
+						Length: 9862
+					^Content/d2k/GAMESFX/O_SCONF1.AUD:
+						Offset: 20176868
+						Length: 18149
+					^Content/d2k/GAMESFX/OI_2MIN.AUD:
+						Offset: 20195017
+						Length: 14697
+					^Content/d2k/GAMESFX/OI_3MIN.AUD:
+						Offset: 20209714
+						Length: 17316
+					^Content/d2k/GAMESFX/OI_4MIN.AUD:
+						Offset: 20227030
+						Length: 14869
+					^Content/d2k/GAMESFX/OI_5MIN.AUD:
+						Offset: 20241899
+						Length: 29873
+					^Content/d2k/GAMESFX/OI_ABORT.AUD:
+						Offset: 20271772
+						Length: 16335
+					^Content/d2k/GAMESFX/OI_ATACK.AUD:
+						Offset: 20288107
+						Length: 20565
+					^Content/d2k/GAMESFX/OI_BDRDY.AUD:
+						Offset: 20308672
+						Length: 25497
+					^Content/d2k/GAMESFX/HI_SELL.AUD:
+						Offset: 20334169
+						Length: 17142
+					^Content/d2k/GAMESFX/OI_DPLOY.AUD:
+						Offset: 20351311
+						Length: 20120
+					^Content/d2k/GAMESFX/AI_PLACE.AUD:
+						Offset: 20371431
+						Length: 29921
+					^Content/d2k/GAMESFX/OI_GANEW.AUD:
+						Offset: 20401352
+						Length: 14467
+					^Content/d2k/GAMESFX/OI_GLOAD.AUD:
+						Offset: 20415819
+						Length: 10906
+					^Content/d2k/GAMESFX/AI_MWIN.AUD:
+						Offset: 20426725
+						Length: 25959
+					^Content/d2k/GAMESFX/AI_NEWOP.AUD:
+						Offset: 20452684
+						Length: 34175
+					^Content/d2k/GAMESFX/AI_NROOM.AUD:
+						Offset: 20486859
+						Length: 30124
+					^Content/d2k/GAMESFX/AI_ORDER.AUD:
+						Offset: 20516983
+						Length: 21881
+					^Content/d2k/GAMESFX/AI_UNRDY.AUD:
+						Offset: 20538864
+						Length: 20567
+					^Content/d2k/SOUND.RS:
+						Offset: 20559431
+						Length: 1929247
+		d2k-linux: Dune 2000 (English)
+			IDFiles:
+				music/ambush.aud: bd5926a56a83bc0e49f96037e1f909014ac0772a
+				setup/setup.z: 937f5ceb1236bf3f3d4e43929305ffe5004078e7
+			Install:
+				copy: MOVIES
```

Comment:
It is lower case `movies` here for whatever reason.

```
Copying /run/media/matthias/DUNE2000/MOVIES/a_br01_e.vqa -> /home/matthias/.openra/Content/d2k/Movies/A_BR01_E.VQA
System.IO.FileNotFoundException: /run/media/matthias/DUNE2000/MOVIES/a_br01_e.vqa does not exist
File name: '/run/media/matthias/DUNE2000/MOVIES/a_br01_e.vqa'
  at System.IO.File.Copy (System.String sourceFileName, System.String destFileName, Boolean overwrite) [0x001bd] in /home/abuild/rpmbuild/BUILD/mono-4.4.0/mcs/class/corlib/System.IO/File.cs:112 
  at System.IO.File.Copy (System.String sourceFileName, System.String destFileName) [0x00000] in /home/abuild/rpmbuild/BUILD/mono-4.4.0/mcs/class/corlib/System.IO/File.cs:72 
  at OpenRA.Mods.Common.Widgets.Logic.InstallFromDiscLogic+<InstallFromDisc>c__AnonStorey5.<>m__1 () <0x40fea6a0 + 0x003ef> in <filename unknown>:0 
Deleting /home/matthias/.openra/Content/d2k/Movies/A_BR01_E.VQA
```

![image](https://cloud.githubusercontent.com/assets/756669/16054604/297d7a7e-326e-11e6-9db3-350ffba90e6e.png)